src/components/views/settings/encryption/ResetIdentityPanel.tsx

```diff
@@ -35,11 +35,16 @@ interface ResetIdentityPanelProps {
      * "compromised" is shown when the user chooses 'reset' explicitly in settings, usually because they believe their
      * identity has been compromised.
      *
+     * "sync_failed" is shown when the user tried to recover their identity but the process failed, probably because
+     * the required information is missing from recovery.
+     *
      * "forgot" is shown when the user has just forgotten their passphrase.
```

Comment:
feels like maybe this documentation belongs next to the definition of `ResetIdentityPanelVariant` now, but :man_shrugging: